CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Bug fixes
 
+* Fixed a crash that did not treat an existing `RealmResult` as an empty list when its originating `RealmObject` class was removed (#2953).
```

Comment:
* Fixed a crash caused by RealmResults if the underlying schema definition is removed. It is now instead treated as a empty list instead.